packages/mdc-tabs/tab-bar/index.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import MDCComponent from '@material/base/component';
+
+import {MDCTab} from '../tab';
+import {strings} from './constants';
+import MDCTabBarFoundation from './foundation';
+
+export {MDCTabBarFoundation};
+
+export class MDCTabBar extends MDCComponent {
+  static attachTo(root) {
+    return new MDCTabBar(root);
+  }
+
+  get tabs() {
+    return this.tabs_;
+  }
+
+  get activeTab() {
+    return this.tabs[this.foundation_.activeTabIndex_];
```

Comment:
I think that will result in a circular reference with `get activeTabIndex`. What might be required is a public method like `getActiveTabIndex` in the foundation which calls `activeTabIndex_`.